### PR TITLE
Release 21.03.0

### PR DESCRIPTION
**Pull request by philippotto**
- X ] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [X] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [X] Ready for review

### CODE REVIEW

File: CHANGELOG.released.md [hunk 1]
```diff
@@ -7,6 +7,44 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Added the possibility to track events for analytics in the backend. [#5156](https://github.com/scalableminds/webknossos/pull/5156)
+
+### Changed
+- Change the font to [Titillium Web](http://nta.accademiadiurbino.it/titillium/). [#5161](https://github.com/scalableminds/webknossos/pull/5161)
+- Make the isosurface feature in the meshes tab more robust. If a request fails, a retry is initiated. [#5102](https://github.com/scalableminds/webknossos/pull/5102)
+- Support for the old invite links was removed. These contained the organization name in the URL. The new links contain a token (can be generated in the users view). For instances with a single organization the old invite links should still work. [#5091](https://github.com/scalableminds/webknossos/pull/5091)
+- Users are no longer allowed to deactivate their own accounts.  [#5070](https://github.com/scalableminds/webknossos/pull/5070)
+- A user needs to confirm his choice if he really wants to leave the dataset upload view while it's still loading. [#5051](https://github.com/scalableminds/webknossos/pull/5049)
```
Comment (fm3) [inline CHANGELOG.released.md:29]:
```suggestion
- Users need to confirm the choice if they really want to leave the dataset upload view while it’s still uploading. [#5051](https://github.com/scalableminds/webknossos/pull/5049)
```

File: CHANGELOG.released.md [hunk 2]
```diff
@@ -7,6 +7,44 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Mailer now uses only TLS1.2 instead of JDK default. [#5138](https://github.com/scalableminds/webknossos/pull/5138)
+- Make keyboard-driven movement through the dataset better aligned with the configured move value in certain scenarios. [#5201](https://github.com/scalableminds/webknossos/pull/5201)
+- User experienced domains are now separated by organization. [#5149](https://github.com/scalableminds/webknossos/pull/5149)
```
Comment (fm3) [inline CHANGELOG.released.md:32]:
```suggestion
- User experience domains are now separated by organization. [#5149](https://github.com/scalableminds/webknossos/pull/5149)
```

File: CHANGELOG.released.md [hunk 3]
```diff
@@ -7,6 +7,44 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Support for KNOSSOS cubes data format was removed. Use the [webKnossos cuber](https://github.com/scalableminds/webknossos-cuber) tool to convert existing datasets saved as KNOSSOS cubes. [#5085](https://github.com/scalableminds/webknossos/pull/5085)
+
+### Added
+- Added an explicit `/signup` (or `/auth/signup`) route. [#5091](https://github.com/scalableminds/webknossos/pull/5091)
+- Added the annotation option "center new nodes" to switch whether newly created nodes should be centered or not. [#4150](https://github.com/scalableminds/webknossos/pull/5112)
+- For webKnossos maintenance, superUsers can now join organizations without being listed as a user there. [#5151](https://github.com/scalableminds/webknossos/pull/5151)
+- Added the possibility to track events for analytics in the backend. [#5156](https://github.com/scalableminds/webknossos/pull/5156)
+
+### Changed
+- Change the font to [Titillium Web](http://nta.accademiadiurbino.it/titillium/). [#5161](https://github.com/scalableminds/webknossos/pull/5161)
```
Comment (fm3) [inline CHANGELOG.released.md:25]:
```suggestion
- Changed the font to [Titillium Web](http://nta.accademiadiurbino.it/titillium/). [#5161](https://github.com/scalableminds/webknossos/pull/5161)
```

File: CHANGELOG.released.md [hunk 4]
```diff
@@ -7,6 +7,44 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Make the isosurface feature in the meshes tab more robust. If a request fails, a retry is initiated. [#5102](https://github.com/scalableminds/webknossos/pull/5102)
```
Comment (fm3) [inline CHANGELOG.released.md:26]:
```suggestion
- Made the isosurface feature in the meshes tab more robust. If a request fails, a retry is initiated. [#5102](https://github.com/scalableminds/webknossos/pull/5102)
```

**Review by fm3 — APPROVED** [review]
See below for some suggestions on wording :)

**Comment by philippotto** [comment]
Thank you :pray:

**Pull request merged**